jsonrpc/server.go

```diff
@@ -397,7 +397,9 @@ func isNil(i any) bool {
 }
 
 func (s *Server) handleRequest(ctx context.Context, req *Request) (*response, error) {
+	s.log.Tracew("Received request", "req", req)
 	if err := req.isSane(); err != nil {
+		s.log.Tracew("Request sanity check failed", "error", err.Error())
```

Comment:
1. no need to call `.Error()` on error interface - logger will do it automatically
2. let's rename "error" to "err" (in our codebase "err" used ~ 54 times in logs while "error" just twice)